doc/AUTHORS

```diff
@@ -91,6 +91,7 @@ improvements to the dbus plugin
 
 
 Peter Nirschl <peter.nirschl@gmail.com>
-development of the crypto plugin, minor bug fixes
+development of the crypto plugin, minor bug fixes, BSD/OS X support
+Github User: petermax2
```

Comment:
Can you write `@petermax2`
I think most people are used to the @mention syntax.